### PR TITLE
add(platform): new doc for pro platform

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes
Add an article on Pro Platform and fix snippet for the Pro badge.

## Checklist

- [X] The first paragraph of the page is on one line.
- [X] The other lines have a line break at 90 characters.
- [X] I checked the output.
- [X] I applied the [style guide](../styleguide.md).
- [X] My links start with `/docs/`.

### CODE REVIEW

**Comment by staceysalamon-aiven** [comment]
Reviewed and approved by PM in devportal.

File: docs/platform/howto/pro-platform.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<EarlyBadge/>
+
+Pro Platform gives enterprise organizations added capabilities for implementing security resilience, reliability, and cost-efficiency strategies for their data infrastructure.
+
+Pro Features are available across disaster recovery, security and compliance,
+advanced networking, and user lifecycle management.
+
+To enable Pro Platform for your organization and use Pro features, contact the sales team
+at sales@aiven.io. You must be a super admin to request this.
```
Comment (ArthurFlageul) [inline docs/platform/howto/pro-platform.md:15]: We should collectively agree on the "contact the sales team" syntax: I've shortened things in many places to `[contact the Sales team](mailto:sales@aiven.io)` to reduce word count and avoid having email addresses in the content, wdyt? 

Also perhaps you can use your new admonition here (or maybe it's not applicable, but if we can, why not)?

Comment (staceysalamon-aiven) [reply docs/platform/howto/pro-platform.md:15]: Yep, totally fine with that. Perhaps another use case for a snippet so we all use the same thing every time?

Regarding the admonition: yes, that was plan once that PR gets merged. 👍

**Review by ArthurFlageul — APPROVED** [review]

**Comment by staceysalamon-aiven** [comment]
> @staceysalamon-aiven you should be able to use the limited access snippet on this branch, now 👍

You mean the super admin admonition, right? It's already been added. :) (And the EA badge is also already there, if that's what you were referring to.)

**Pull request closed without merging**